app/components/school_information/your_schools/participants_school_search_component.html.erb

```diff
@@ -0,0 +1,38 @@
+<div class="govuk-grid-row">
+  <%= form_with model: @school_search_form, url: lead_providers_your_schools_path, builder: GOVUKDesignSystemFormBuilder::FormBuilder, method: :post do |f| %>
```

Comment:
Was there any reason why you had to explicitly specify builder: GOVUKDesignSystemFormBuilder::FormBuilder

Reply:
Maybe components are compiled before we set the formbuilder in our Rails app? Or independently of it? Maybe? 

That would probably be a very good reason not to put a form in a component in my view.

Reply:
Yes - this came as a complete surprise - in normal form in views - because it's set up in the application controller - we will use the Gov form builder and it works like magic. What really surprised me that the gov form builder methods were not available when I refactored the solution to components. It took me a while to discover what was the problem - and that when using components you have the explicitly specify that you are using Gov form builder. The number of swearwords I've used was directly correlated to this.

Reply:
Is there a way to configure all view components to use this form builder? 

I really hate the idea of specifying form builder every time we want to put a firm in a component.

Reply:
Specified Base Component and put a method wrapping form builder param inside